dynamic_instrumentation/src/main.rs

```diff
@@ -59,6 +60,10 @@ struct Args {
     #[clap(long)]
     set_runtime: bool,
 
+    /// `$RUSTFLAGS`, but as a CLI argument, not an inheritable environment variable.
```

Comment:
This should explain the use case for this option, not just what it is. This is doubly confusing because the implementation actually is setting an inheritable environment variable.

Reply:
This should also document that the argument to this CLI flag is appended (after a space) to the value of `RUSTFLAGS` if it is set in the environment.

Reply:
Okay, I'll add much more thorough documentation.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/629/commits/aebde0a35bc7236e4223d44f7028178b139da089.